pkg/limayaml/validate.go

```diff
@@ -18,41 +18,63 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func validateFile(f File, fieldName string) error {
+	if !strings.Contains(f.Location, "://") {
+		if _, err := localpathutil.Expand(f.Location); err != nil {
+			return fmt.Errorf("field `%s.location` refers to an invalid local file path: %q: %w", fieldName, f.Location, err)
+		}
+		// f.Location does NOT need to be accessible, so we do NOT check os.Stat(f.Location)
+	}
+	switch f.Arch {
+	case X8664, AARCH64, RISCV64:
+	default:
+		return fmt.Errorf("field `%s.arch` must be %q or %q, got %q", fieldName, X8664, AARCH64, f.Arch)
+	}
+	if f.Digest != "" {
+		if !f.Digest.Algorithm().Available() {
+			return fmt.Errorf("field `%s.digest` refers to an unavailable digest algorithm", fieldName)
+		}
+		if err := f.Digest.Validate(); err != nil {
+			return fmt.Errorf("field `%s.digest` is invalid: %s: %w", fieldName, f.Digest.String(), err)
+		}
+	}
+	return nil
+}
+
 func Validate(y LimaYAML, warn bool) error {
 	switch *y.Arch {
-	case X8664, AARCH64:
+	case X8664, AARCH64, RISCV64:
 	default:
-		return fmt.Errorf("field `arch` must be %q or %q , got %q", X8664, AARCH64, *y.Arch)
+		return fmt.Errorf("field `arch` must be %q, %q or %q, got %q", X8664, AARCH64, RISCV64, *y.Arch)
```

Comment:
Nitpicking: mixing oxford comma:

```suggestion
		return fmt.Errorf("field `arch` must be %q, %q, or %q; got %q", X8664, AARCH64, RISCV64, *y.Arch)
```

Reply:
fixed